lib/semian/simple_exponential_smoother.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Semian
+  # SimpleExponentialSmoother implements Simple Exponential Smoothing (SES) for forecasting
+  # a stable baseline error rate in adaptive circuit breakers.
+  #
+  # SES focuses on the level component only (no trend or seasonality), using the formula:
+  #   smoothed = alpha * value + (1 - alpha) * previous_smoothed
+  #
+  # Key characteristics:
+  # - Adaptive alpha adjusts based on confidence period and convergence direction
+  # - Converges faster when error rate is decreasing (conservative)
+  # - Converges slower when error rate is increasing (cautious)
+  # - Drops extreme values above cap to prevent outliers from distorting the forecast
+  #
+  class SimpleExponentialSmoother
+    DEFAULT_ALPHA = 0.001
```

Comment:
we never really use the default alpha or the inital alpha. We are always setting it to one of the other values. So let's remove it?